hive-llap/test_llap.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from absl.testing import absltest
+from absl.testing import parameterized
+
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+
+class LLAPTestCase(DataprocTestCase):
+    COMPONENT = 'hive-llap'
+    INIT_ACTIONS = ['hive-llap/llap.sh']
+    TEST_SCRIPT_FILE_NAME = 'run_hive_commands.py'
+    OPTIONAL_COMPONENTS = ["ZOOKEEPER"]
+
+    def verify_instance(self, name):
+        self.upload_test_file( 
+            os.path.join(
+                os.path.dirname(os.path.abspath(__file__)),
+                self.TEST_SCRIPT_FILE_NAME), name)
+        self.__run_test_script(name)
+        self.remove_test_script(self.TEST_SCRIPT_FILE_NAME, name)
+
+    def __run_test_script(self, name):
+        self.assert_instance_command(
+            name, "python {}".format(self.TEST_SCRIPT_FILE_NAME))
+
+    @parameterized.parameters(
+        ("HA", ["m-0"]),
+        ("STANDARD", ["m"])
+        )
+
+    def test_llap(self, configuration, machine_suffixes):
+        if self.getImageOs() == 'centos':
+            self.skipTest("Not supported in CentOS-based images")
+
+        self.createCluster(configuration, 
+                            self.INIT_ACTIONS, 
+                            metadata=None,
```

Comment:
I think there's no need to mention parameters with None value. You can skip those.

Reply:
done